tutorials/e2e-huggingface-summarization/src/run_summarization/run.py

```diff
@@ -0,0 +1,377 @@
+import logging
```

Comment:
I suggest aligning file names as if "src" was some team's code base. For instance:
- rename "run_summarization" by "summarization"
- rename "run.py" by "train.py"
- maybe rename "baseline.py" by "baseline_evaluation.py" ?